sdk/iOS/test/MSSyncTableTests.m

```diff
@@ -1878,54 +2218,6 @@ -(void) testIncrementalPullWithOrderByFails
     XCTAssertEqual(1, filter.actualRequests.count);
 }
 
-- (void)testPullResultingInErrorAllowsDependentOperationsToComplete
```

Comment:
This test verifies that the pull NSOperation is in finished state if the push fails. There is just one of many cases (error at different points in the workflow and not just during push, no error case, etc) in which we need to test for successful completion of the NSOperation. All this is being already covered as part of the 2 push and pull UTs added.